courses/README.md

```diff
@@ -0,0 +1,44 @@
+xPRO Programs, Courses and Course Runs
+
+**SECTIONS**
+1. [How to create a new courserun](#how-to-create-a-new-courserun)
+
+## How to create a new courserun
+
+The course team will announce on the xpro_newcourses moira list, after the course has been created in edX Studio. The 
+announcement should include at minimum the course id. Pricing information will be provided by the marketing team, usiing 
+the same moira list. 
+
+1. Create a program, if necessary
+
+  1. Link to How to create a program
+
+2. Create a course if necessary 
+
+  1. Link to How to create a course
+  
+3. Create a courserun at /admin/courses/courserun/add/
+
+  - the run tag should be the last component of the course id, i.e. R4
+  - assuming this is an open edX course, the courseware path should be of the form `/courses/{course id}/course/`. Note that
+    this path is not validated. (See https://github.com/mitodl/mitxpro/issues/1667)
+  - the courserun title and dates will be pulled in automatically from Studio on a nightly basis. Or you can use the 
+    management command `sync_courseruns` to do it immediately. 
+  - Click save and note the courserun id, you will need it later. 
```

Comment:
Minor, but might be nice to run through each of the items in the form in the order they appear: 

```suggestion
  - Course: xyz
  - Title: xyz
  - Courseware id: xyz
  - etc...
```